pkgs/tools/misc/direnv/default.nix

```diff
@@ -1,24 +1,28 @@
-{ stdenv, fetchFromGitHub, buildGoPackage, bash}:
+{ stdenv, fetchFromGitHub, buildGoPackage, bash, fetchpatch }:
 
 buildGoPackage rec {
   name = "direnv-${version}";
-  version = "2.17.0";
+  version = "2.18.1";
   goPackagePath = "github.com/direnv/direnv";
 
   src = fetchFromGitHub {
     owner = "direnv";
     repo = "direnv";
     rev = "v${version}";
-    sha256 = "1dmanqpifx27cz41yc3ijpij0wrbgw9qny2d4n6jppfwf2qzyq4s";
+    sha256 = "1x5yx29wjkx7z9b0bfkw8kncrqyybkwan8c6ckpbnvm2ad057p17";
   };
 
   postConfigure = ''
     cd $NIX_BUILD_TOP/go/src/$goPackagePath
   '';
 
-  buildPhase = ''
-    make BASH_PATH=${bash}/bin/bash
-  '';
+  patches = [
+    ./build.patch
```

Comment:
https://github.com/direnv/direnv/pull/417

Reply:
master now has direnv v2.18.2 which includes the patch